maven_install.json

```diff
@@ -1,6 +1,6 @@
 {
     "dependency_tree": {
-        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": -1332813254,
+        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": -790274667,
```

Comment:
Do we have anything in the documentation that shows how to regenerate this `maven_install` file?   Again, doesn't need to be done in this PR.  But would be helpful to know more about it.

Reply:
Where would be a good spot to add it? Currently I only added it here in the `WORKSPACE` file. https://github.com/apache/incubator-heron/blob/ee73293b2ce5f1c0aa0506a4398bb2efe696944f/WORKSPACE#L152-L155